OTVision/detect/yolo.py

```diff
@@ -170,7 +170,7 @@ def detect_video(
         )
         batch_no += 1
 
-        pbar.update(1)
+        pbar.update(chunksize)
```

Comment:
should be `pbar.update(len(rgb_transformed_batch))` to fit size of last batch?

Reply:
Yes, otherwise pbar.close() is necessary at the end.